package.json

```diff
@@ -44,7 +44,12 @@
       ]
     },
     "eslint": {
-      "command": "eslint --cache --ext .js,.ts --fix ."
+      "command": "eslint --cache --ext .js,.ts --fix .",
+      "files": [
+        ".eslintignore",
+        ".eslintrc.js",
+        "src/**/*.ts"
```

Comment:
Note: because of how `wireit` in watch mode this will run 2 times. 
I remember need to explicitly set this for VSCode to fix EsLint errors.
```
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
  ```

Reply:
Thanks! This should be non-intrusive, though. I use `--watch`, but Maksim does not. If one does not use `--watch`, this is a no-op. We also have that snippet in our vscode config: https://github.com/GoogleChromeLabs/chromium-bidi/blob/a09dc1c3132a074710632decb05f6cbc8cac29b7/.vscode/settings.json#L6